### PR TITLE
connected endpoints from users microservice to the gateway

### PR DESCRIPTION
**Pull request by Kar977**

### CODE REVIEW

File: auth/auth0_client.py [hunk 1]
```diff
@@ -31,53 +33,93 @@ def verify_access_token(token: str):
...
+    await get_current_user(request)
+
+    return {"view": "private"}
+
+
+@router.get("/private/roles")
+async def private_roles(request: Request):
+    await check_role(request, ["business-owner"])
+
+    return {"message": "You are able to see it, because you have business-owner role!"}
```
Comment (Kar977) [inline auth/auth0_client.py]: functions from line 114 to 125 deleted (added by accident)

File: routers/customer_manager/visits.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import httpx
 from fastapi import APIRouter, Request
-from fastapi import HTTPException
 from routers.customer_manager.schemas import CreateVisitRequest
+from routers.customer_manager.slots import send_request_to_service
 
+from settings import Settings
 
 router = APIRouter(prefix="/visits")
 MICROSERVICE_URL = "http://localhost:8001"
```
Comment (Kar977) [inline routers/customer_manager/visits.py:8]: deleted

Comment (Kar977) [inline routers/customer_manager/visits.py]: Line 8 deleted

File: routers/customer_manager/workday.py [hunk 3]
```diff
@@ -1,39 +1,50 @@
...
 from fastapi import APIRouter, Request
-from fastapi import HTTPException
-
 from routers.customer_manager.schemas import DeleteWorkdayRequest, CreateWorkdayRequest
+from routers.customer_manager.slots import send_request_to_service
+
+from settings import Settings
 
 router = APIRouter(prefix="/workdays")
 MICROSERVICE_URL = "http://localhost:8001"
```
Comment (Kar977) [inline routers/customer_manager/workday.py]: Line 8 deleted

File: auth/auth0_client.py [hunk 4]
```diff
@@ -62,26 +62,26 @@ async def verify_access_token(token: str) -> TokenData:
         )  # ToDo change detailed information on general info before release on production
 
 
-async def get_current_user(request: Request) -> TokenData:
-    token = request.session.get("access_token")
+async def get_current_user(request: Request):  # -> TokenData:
```
Comment (kgarbacinski) [inline auth/auth0_client.py:65]: zbędny komentarz

**Review by kgarbacinski — APPROVED** [review]

**Pull request merged**